src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles'
+import { Modal as MaterialModal } from '@material-ui/core'
+
+function getModalStyle() {
+    const top = 30
+    const left = 30
+
+    return {
+        top: `${top}%`,
+        left: `${left}%`,
+        transform: `translate(-${top}%, -${left}%)`,
+    }
+}
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        paper: {
+            position: 'absolute',
+            width: 400,
+            backgroundColor: theme.palette.background.paper,
+            border: '2px solid #000',
+            boxShadow: theme.shadows[5],
+            padding: theme.spacing(2, 4, 3),
+        },
+    })
+)
+
+export default function Modal({ isOpened, children, onClose }) {
+    const classes = useStyles()
+    const [modalStyle] = React.useState(getModalStyle)
+
+    const body = (
```

Comment:
why moving the body in a variable?

Reply:
that comes from [material-ui SimpleModal](https://material-ui.com/components/modal/#simple-modal)